src/f2py/adflow.pyf

```diff
@@ -65,6 +65,17 @@ python module libadflow
          subroutine releasememorypart2
          end subroutine releasememorypart2
 
+         subroutine getmonitorvariablenames(nvar, monitor_variables)
+          integer(kind=inttype), intent(in) :: nvar
+          character, dimension(nvar,32), intent(out):: monitor_variables
+        end subroutine getmonitorvariablenames
```

Comment:
VSCode messes it up for me all the time. Is there a way to set it up for Fortran?

Reply:
Not that I have found, but keep us posted if you find anything.

Reply:
I originally wanted to propose that we format our Fortran code the same way as our Python stuff, using for example [fprettify](https://github.com/pseewald/fprettify) which is already integrated into the Modern Fortran vs code extension. But I didn't want to push on this since I don't do a lot of Fortran development.